### ISSUE TITLE
Missing Dependencies of useEffect Hooks

### ISSUE THREAD

**Issue by shubhanshu02** [COPS-IITBHU/hackalog-frontend#69]
### Description

In some files, the `useEffect` has missing dependencies. The issue proposes including such unmet dependencies in the dependencies array of the `useEffect` hook.

### Occurrences

File: `pages/hackathon/[slug]/teams/[team_id]/submit.js`
Line: 57
Missing Dependency: `token`

File: `pages/hackathon/[slug]/index.js`
Line: 77
Missing Dependency: `hackathon`

**Comment by shubhanshu02** [comment]
Yes, @Devansh-gaur-1611. You are welcome to put up a PR for that.

**Comment by Devansh-gaur-1611** [comment]
Thanks, @shubhanshu02 bhaiya for assigning me. I have started working on this issue and will report you about progress in some time.
